### PR TITLE
iOS-3280 Core | add test build with xcode 16 on pr

### PR DESCRIPTION
**Pull request by mgolovko**

### CODE REVIEW

File: .github/workflows/automerge-from-actions.yaml [hunk 1]
```diff
@@ -1,16 +0,0 @@
-name: Auto-merge-actions
```
Comment (mgolovko) [inline .github/workflows/automerge-from-actions.yaml:1]: unused

File: .github/workflows/build_test_ios_16.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: Run tests on PR iOS 16
```
Comment (joe-pusya) [inline .github/workflows/build_test_ios_16.yml:1]: Xcode 16?

File: .github/workflows/build_test_ios_16.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      - name: Checkout
+        uses: actions/checkout@v3
+      
+      - name: Prepare deps
+        uses: ./.github/actions/prepare-deps
+
+      - name: Setup Xcode
+        uses: maxim-lobanov/setup-xcode@v1.5.1
+        with:
+          xcode-version: '15.4'
```
Comment (joe-pusya) [inline .github/workflows/build_test_ios_16.yml:23]: Xcode 16?

File: .github/workflows/build_test_ios_16.yml (hunk 2 above)
Comment (mgolovko) [reply .github/workflows/build_test_ios_16.yml]: Fixed!

**Pull request merged**